src/com/owncloud/android/ui/SquareImageView.java

```diff
@@ -0,0 +1,25 @@
+package com.owncloud.android.ui;
```

Comment:
Copyright and license comments are needed here.

@tobiasKaminsky , did you write these files, or are copied from other place?